codehawk.properties

```diff
@@ -8,7 +8,7 @@ defaults.mavenGroupId= org.codehawk.plugin
 defaults.mavenArtifactId= codehawk
 
 1.4.description= Added new rules
-1.4.sqVersions=[7.9,LATEST]
+1.4.sqVersions=[7.9,8.4.*]
```

Comment:
When you replace `LATEST` on the version you're archiving, you should always replace it with the current version. Otherwise your users on the latest SQ version won't be prompted in-app to upgrade to _your_ latest version.